sources/packages/web/src/views/student/StudentProfileEdit.vue

```diff
@@ -54,6 +54,7 @@ export default {
         dateOfBirth: studentInfo.birthDateFormatted,
         mode: StudentProfileFormModes.StudentEdit,
         identityProvider: AuthService.shared.userToken?.IDP,
+        sinConsent: studentInfo.sinConsent,
```

Comment:
I believe that this value is already present in the `studentInfo` that is associated in line 50, right?

Reply:
Correct, I will remove it.